httpx/concurrency/asyncio.py

```diff
@@ -80,11 +79,15 @@ def set_write_timeouts(self) -> None:
         self.raise_on_write_timeout = True
 
 
-class Reader(BaseReader):
+class Stream(BaseStream):
     def __init__(
-        self, stream_reader: asyncio.StreamReader, timeout: TimeoutConfig
-    ) -> None:
+        self,
+        stream_reader: asyncio.StreamReader,
```

Comment:
A couple `DeprecationWarning`s show up in Travis:
```
DeprecationWarning: StreamReader is deprecated since Python 3.8 in favor of Stream, and scheduled for removal in Python 3.10
DeprecationWarning: StreamWriter is deprecated since Python 3.8 in favor of Stream, and scheduled for removal in Python 3.10
```
Should those be addressed in this PR?

Reply:
Good catch, thanks! :+1: I believe this is independant (warnings also show up on the master build), so it can be addressed in a separate PR — I tracked it in #256.